pom.xml

```diff
@@ -128,7 +128,9 @@
     <findbugs.version>3.0.5</findbugs.version>
     <!-- surefire/failsafe plugin option -->
     <forkCount>1</forkCount>
+    <hadoop.profile>2</hadoop.profile>
```

Comment:
I think this will actually disable the hadoop2 profile, because of `!hadoop.profile`. Because of https://issues.apache.org/jira/browse/MNG-3328 , when we had a 1 and 2 profile, we had to create a third hadoop default profile which had `!hadoop.profile` and was identical (duplicated) to the hadoop2 profile. It was tedious, but I think it might be the only workaround to MNG-3328.

Reply:
There is a build extension mentioned in the comments of that issue which could be a good workaround to creating a third profile. It provides a mechanism to add a scripted expression to determine if a profile should be active or not. It looks promising, but I'd want to test it in Eclipse M2E (and maybe other IDEs which support Maven) first to see if the correct profile activates by default there also.

Reply:
> when we had a 1 and 2 profile, we had to create a third hadoop default profile

I remembered this, but didn't look back in history to figure it out again. Let me re-add that. Would be better than having `-Dhadoop.profile=2` oddly not work.